ci/Jenkinsfile

```
@@ -0,0 +1,108 @@
+#!/usr/bin/env groovy
+library 'status-jenkins-lib@v1.7.13'
+
+pipeline {
+    agent {
+        label 'linux && x86_64 && qt-5.15.2'
+    }
+
+    environment {
+        SQUISH_DIR = "/opt/squish-runner-7.1-20230222-1555"
+        PYTHONPATH = "${SQUISH_DIR}/lib:${SQUISH_DIR}/lib/python:${PYTHONPATH}"
+        LD_LIBRARY_PATH = "${SQUISH_DIR}/lib:${SQUISH_DIR}/python3/lib:${LD_LIBRARY_PATH}"
+    }
+
+    parameters {
+        string(
+          name: 'BRANCH',
+          description: 'Paste a branch name from test repository.',
+          defaultValue: 'master'
+        )
+        string(
+          name: 'BUILD',
+          description: 'Paste a number of build that you want to test.',
+          defaultValue: 'PR-'
+        )
+        string(
+          name: 'FILTER',
+          description: 'Paste "-k" with test name/part of test name to run specific test or/and "-m" with tag to run specific scope of tests',
```

please , add marks and their description to pytest.ini.  For example
```suggestion
         markers = 
             all: to run all the tests
             wallet: to run only wallet tests
             settings: to run only settings tests
```

---

Wouldn't make sense to separate this into two separate parameters? `TEST_SCOPE` and `TEST_NAME`?
By expecting users to remember they need to use flags in the value you're just creating friction.

---

Fixed

---

About markers, they should be added first in tests, for now, we have only the smoke tag and it is added in pytest